client/src/partials/patients/documents/views/documentView.js

```diff
@@ -0,0 +1,44 @@
+angular.module('bhima.controllers')
+.controller('DocumentViewController', DocumentViewController);
+
+DocumentViewController.$inject = [
+  'ModalService', '$state', 'DocumentService', 'Upload', 'NotifyService'
+];
+
+function DocumentViewController(Modal, $state, Document, Upload, Notify) {
+  var vm = this;
+
+  /** global objects */
```

Comment:
Just a note - for single line comments, please use single-line comment syntax like
```js
// this is a comment
```
instead of multi-line comment syntax
```js
/* this is difficult to change later */
/** so is this one! */
```

Please reserve `/** */` comments for documentation comments that describe important properties, methods, APIs and parameters.  JSDoc will try to parse these comments and make very strange documentation. [See this link for an example](https://doclets.io/IMA-WorldHealth/bhima-2.X/master#dl-agent).